Repo-Integration/Self-Hosted/setup.sh

```diff
@@ -77,6 +81,64 @@ echo "${grn}Download Success!!!${end}"
 
 }
 
+function cert_add(){
```

Comment:
There's a better way to incorporate certs into the containers that require no changes to the dockerfiles.  I don't support this approach.